Cython/Compiler/ExprNodes.py

```diff
@@ -1928,6 +1929,7 @@ class NameNode(AtomicExprNode):
     allow_null = False
     nogil = False
     inferred_type = None
+    annotation = None
```

Comment:
I don't think we need to define it here. Every `ExprNode` has an `annotation` attribute.